lib/round.rb

```diff
@@ -0,0 +1,96 @@
+require './lib/deck'
+
+class Round
+  attr_reader :deck,
+              :turns
+  attr_accessor :correct_answers,
+                :turn_count,
+                :total_cards,
+                :categories
```

Comment:
It doesn't look like you're trying to update these values outside of this class, so for that reason, these do not need to be attr_accessors.